web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -694,6 +694,25 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                 })}
               </Select>
             </Form.Item>
+            <Form.Item
+              label={formatMessage({ id: 'page.upstream.step.scheme' })}
+              name="scheme"
+            >
+              <Select disabled={readonly}>
```

Comment:
```suggestion
              <Select disabled={readonly}>
              {['http', 'https', 'grppc', 'grpcs'].map( item => (  
                  <Select.Option value={item}>
                     {item}
                   </Select.Option>))
              </Select>
```

Reply:
use `index` in `loop`

Reply:
you mean replace `item` to `index`?

Reply:
![image](https://user-images.githubusercontent.com/2106987/113088650-a2046700-9218-11eb-94bf-045cb0759c8a.png)

here :)